bbm-0.4.0/bbm-0.4.0.js

```diff
@@ -0,0 +1,49 @@
+/* Better Bootstrap Modals (bbm) v0.4.0
+** https://github.com/danielpox/better-bootstrap-modals
+** Copyright (c) 2016 Daniel Pox
+** Contibutor: Mouse0270 https://github.com/mouse0270
+** Released under The MIT License (MIT)
+==========================================================*/
+// Get Script Data Attributes
+var dataVariables = (function() { var scripts = document.getElementsByTagName("script"); return scripts[scripts.length - 1].dataset; })();
+$(document).ready(function() {
+    var $modalContainer = $('<div class="modal-container"></div>');
+
+    // Check if `.modal-container` exists.
+    // If so, set $modalContainer to first instance
+    // Otherwise add $modalContainer to `body`
+    if ($('body .modal-container').length >= 1) {
+    	$modalContainer = $('.modal-container').eq(0)
+    }else{
+		$('body').append($modalContainer);
+	}
+
+	// If show is set to first, add class to `.modal-container` to limit modals to only show one at a time
+	if (dataVariables.show == 'first') {
+		$modalContainer.addClass('show-first-only');
+	}
+
+	// Add Opening modals to `.modal-container`
+    $('.modal').on('show.bs.modal', function() {
+    	// Check to see if forceBackdrop is set, If not use default backdrop setting
+    	var forceBackdrop = (typeof dataVariables.forceBackdrop != 'undefined' ? (dataVariables.forceBackdrop === 'true') : $(this).data('bs.modal').options.backdrop);

```

Comment:
This lines checks to see if the user decided to use `data-force-backdrop` and if so it uses the value within the `data attribute` otherwise it uses the default one based on the `modal`. Most likely `true` unless they set `data-backdrop` to `false` when they defined the `modal`